make/photon/prepare/templates/registry/config.yml.jinja

```diff
@@ -17,7 +17,7 @@ storage:
     disable: true
 {% endif %}
 redis:
-  addr: {{redis_host}}:{{redis_port}}
+  addr: {{redis_schema}}://{{redis_host}}
```

Comment:
Does registry support this?

Reply:
Same question. Have you confirmed or verified already? @bitsf

Reply:
yes, we are using DialURL()

Reply:
Are you sure `DialURL()` can handle multiple hosts?

Reply:
No, I mean DialURL() is just for legacy redis schema, it can handle redis:// and rediss://
And I have seperate logic to handle redis+sentinel://
Here I just want to use a unified method to input this redis addr